cmake/utilities.cmake

```diff
@@ -192,18 +192,13 @@ function(vcpkg_target_add_warning_options TARGET)
         if(VCPKG_DEVELOPMENT_WARNINGS)
             target_compile_options(${TARGET} PRIVATE -W4)
             if(VCPKG_COMPILER STREQUAL "clang")
-                # -Wno-range-loop-analysis is due to an LLVM bug which will be fixed in a
-                # future version of clang https://reviews.llvm.org/D73007
                 target_compile_options(${TARGET} PRIVATE
                     -Wmissing-prototypes
                     -Wno-missing-field-initializers
```

Comment:
```suggestion
                    -Wno-missing-field-initializers
                    $<$<VERSION_LESS:$<CXX_COMPILER_VERSION>,10.0.0>:-Wno-range-loop-analysis>
```

Reply:
This is already guarded by VCPKG_DEVELOPMENT_WARNINGS so I'm not sure that kind of defense is reasonable.